code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -273,19 +273,21 @@ Difficulty: Medium
 	dragon_fire_line(src, turfs)
 
 //fire line keeps going even if dragon is deleted
-/proc/dragon_fire_line(source, list/turfs)
+/proc/dragon_fire_line(source, list/turfs, fire_source = "fire breath", ignite_turfs = FALSE, power = 4, flame_color = "red")
```

Comment:
could you refactor this out of here pretty please c:, i can do this if you want.

Reply:
Sure, where would you want me to move it instead?